editor/src/core/workers/common/worker-types.ts

```diff
@@ -8,6 +8,7 @@ import {
   ParsedTextFile,
   ProjectFile,
 } from '../../shared/project-file-types'
+import { emptySet } from '../../shared/set-utils'
```

Comment:
New import but it is not used anywhere as far as I see

Reply:
Fixed